miniserver/config.js

```diff
@@ -0,0 +1,84 @@
+require('dotenv').config()
+const ethers = require('ethers')
+const DEBUG = process.env.MINISERVER_DEBUG === 'true' || process.env.MINISERVER_DEBUG === '1'
+const APIDOCS = process.env.APIDOCS === 'true' || process.env.APIDOCS === '1'
+const config = {
```

Comment:
Let's clean this up to bare minimum

Reply:
Have updated @polymorpher can you review again and see what else (if anything) should be removed 🙏